docs/environment-variables.md

```diff
@@ -1,61 +1,73 @@
+Those are not listed here, please consult `graph-node --help` for details on
+those.
 
 ## Getting blocks from Ethereum
 
-* `ETHEREUM_POLLING_INTERVAL`: how often to poll Ethereum for new blocks
-(in ms, defaults to 500ms)
-* `ETHEREUM_RPC_MAX_PARALLEL_REQUESTS`: how many RPC connections to start
-in parallel for block retrieval (defaults to 64)
-* `ETHEREUM_FAST_SCAN_END`: `graph-node` locates
-  blocks with events for a particular subgraph. Most subgraphs do not have
-  events in the first few million blocks, so `graph-node` optimizes for that
-  case by inquiring about a large range. The value of this variable
-  is the block number at which we switch from probing large ranges to
-  ranges of size `ETHEREUM_BLOCK_RANGE_SIZE` (defaults to 4000000).
-* `ETHEREUM_TRACE_STREAM_STEP_SIZE`: `graph-node` queries traces for a given block
-  range when a subgraph defines call handlers or block handlers with a call filter.
-  The value of this variable controls the number of blocks to scan in a single RPC request for traces
-  from the Ethereum node.
+* `ETHEREUM_POLLING_INTERVAL`: how often to poll Ethereum for new blocks (in ms,
+defaults to 500ms)
+* `ETHEREUM_RPC_MAX_PARALLEL_REQUESTS`: how many RPC connections to start in
+parallel for block retrieval (defaults to 64)
+* `ETHEREUM_FAST_SCAN_END`: `graph-node` locates blocks with events for a
+  particular subgraph. Most subgraphs do not have events in the first few
+  million blocks, so `graph-node` optimizes for that case by inquiring about a
+  large range. The value of this variable is the block number at which we switch
+  from probing large ranges to ranges of size `ETHEREUM_BLOCK_RANGE_SIZE`
+  (defaults to 4000000).
+* `ETHEREUM_TRACE_STREAM_STEP_SIZE`: `graph-node` queries traces for a given
+  block range when a subgraph defines call handlers or block handlers with a
+  call filter. The value of this variable controls the number of blocks to scan
+  in a single RPC request for traces from the Ethereum node.
 * `DISABLE_BLOCK_INGESTOR`: set to `true` to disable block ingestion. Leave
   unset or set to `false` to leave block ingestion enabled.
-* `ETHEREUM_BLOCK_BATCH_SIZE`: number of Ethereum blocks to request in
-  parallel (defaults to 50)
-* `ETHEREUM_BLOCK_RANGE_SIZE` - number of blocks to scan for events in
-  each request (defaults to 10000).
-* `ETHEREUM_PARALLEL_BLOCK_RANGES` - Maximum number of parallel `eth_getLogs` calls to make when scanning logs for a subgraph. Defaults to 100.
+* `ETHEREUM_BLOCK_BATCH_SIZE`: number of Ethereum blocks to request in parallel
+  (defaults to 50)
+* `ETHEREUM_BLOCK_RANGE_SIZE`: number of blocks to scan for events in each
+  request (defaults to 10000).
+* `ETHEREUM_PARALLEL_BLOCK_RANGES` - Maximum number of parallel `eth_getLogs`
+  calls to make when scanning logs for a subgraph. Defaults to 100.
+* `ETHEREUM_START_BLOCK`: the block number at which subgraphs should start
```

Comment:
It might be good to have a warning along the lines of 'do not use this for production subgraphs as it can cause the subgraph to contain incorrect data' or some such here.

Reply:
Agreed, can also log a runtime warning.